tests/integration/targets/systemd_creds_encrypt/tasks/main.yaml

```diff
@@ -53,3 +53,59 @@
           - '"SetCredentialEncrypted=web: " in pretty_encrypted_secret.value'
         fail_msg: "SetCredentialEncrypted is not in the output"
         success_msg: "SetCredentialEncrypted is in the output"
+
+    - name: Ensure the dest file doesn't already exist
+      ansible.builtin.file:
+        path: /tmp/systemd_cred_encrypt_test_secret.cred
```

Comment:
Instead of `/tmp`, I'd recommend you use `setup_remote_tmpdir` - see other tests that use it as example.